src/AdvancedVI.jl

```diff
@@ -1,52 +1,50 @@
+
 module AdvancedVI
 
-using Random: Random
+using UnPack, Accessors
```

Comment:
IMO it's generally a bad idea to use UnPack: https://github.com/devmotion/SimpleUnPack.jl#readme